sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasQueryParameters.cs

```diff
@@ -0,0 +1,129 @@
+﻿//// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+
+namespace Azure.Storage.Sas
+{
+    /// <summary>
+    /// A <see cref="BlobSasQueryParameters"/> object represents the components
+    /// that make up an Azure Storage Shared Access Signature's query
+    /// parameters.  You can construct a new instance using
```

Comment:
NIT: Extra spaces before `You`

Reply:
I'm a "two spaces after a period" fan.&nbsp;&nbsp;Even if the web tries to hide them by default.